.drone.starlark

```diff
@@ -455,6 +554,53 @@ config = {
 				}
 			],
 		},
+		'core-webui-acceptance-latest-nightly': {
+			'suites': [
+				'webUICore1',
+				'webUICore2',
+			],
+			'databases': [
+				'mysql:5.7',
+			],
+			'servers': [
+				'latest',
```

Comment:
compared to the section above, this new section runs against `latest` rather than `daily-master-qa`